collector/snapshots_test.go

```diff
@@ -50,23 +44,30 @@ func TestSnapshots(t *testing.T) {
 			t.Fatalf("Failed to fetch or decode snapshots stats: %s", err)
 		}
 		t.Logf("[%s] Snapshots Response: %+v", ver, stats)
-		repositoryStats := stats["test1"]
-		snapshotStats := repositoryStats.Snapshots[0]
 
-		if len(snapshotStats.Indices) != 2 {
-			t.Errorf("Bad number of snapshot indices")
-		}
-		if len(snapshotStats.Failures) != int(snapshotStats.Shards.Failed) {
-			t.Errorf("Bad number of snapshot failures")
-		}
-		if snapshotStats.Shards.Total != 10 {
-			t.Errorf("Bad number of snapshot shards total")
-		}
-		if snapshotStats.Shards.Successful != 10 {
-			t.Errorf("Bad number of snapshot shards successful")
-		}
-		if len(repositoryStats.Snapshots) != 1 {
-			t.Errorf("Bad number of repository snapshots")
+		for repo, snapshotResponse := range stats {
+			if repo == "fail" {
+				if snapshotResponse.Snapshots != nil {
+					t.Errorf("Returning non-nil Snapshots response for inaccessible repo")
+				}
+				continue
+			} else {
+				if len(snapshotResponse.Snapshots[0].Indices) != 2 {
```

Comment:
Just wondering why this check hardcoded to 2 (same as Shards).

Does this make it very specific to certain ES deployments?